runtime/lua/vim/lsp.lua

```diff
@@ -594,6 +594,14 @@ function lsp.enable(name, enable)
     })
 end
 
+--- Checks if the given LSP config is enabled or not.
+---
+--- @param name string Config name
+--- @return boolean
+function lsp.is_enabled(name)
+  return lsp._enabled_configs[name] ~= nil
```

Comment:
@lewis6991 will know better but I think that we should also check `lsp.config._configs['*']` here (in case the LSP server was configured with `vim.lsp.config('*', ...)`.

Reply:
`*` is a weird case. I don't think we need to check for it here.